langchain/src/client/langchainplus.ts

```diff
@@ -234,6 +244,63 @@ export class LangChainPlusClient {
     return response.json() as T;
   }
 
+  public async readRun(runId: string): Promise<Run> {
+    return await this._get<Run>(`/runs/${runId}`);
+  }
+
+  public async listRuns({
+    sessionId,
+    sessionName,
+    executionOrder = 1,
+    runType,
+  }: ListRunsParams): Promise<Run[]> {
```

Comment:
I'm being inconsistent here... What would you recommend as the preferred way to specify args?